style.css

```diff
@@ -630,3 +647,430 @@
+    color: #344563;
+  }
+
+  .desktop-menu ul li {
+    margin: 0 1rem 0 1rem;
+    font-size: 15px;
+    font-family: var(--font-family);
+    color: #344563;
+  }
+
+  #first-sec {
+    padding: 7rem 14vw 9rem 14vw;
+    background-image: url("images/Header\ bg@2x.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    height: 28rem;
+    border-bottom-left-radius: 100px;
+    background-color: #fff;
+    margin-bottom: 4rem;
+  }
+
+  #first-sec .mobile-h1 {
+    display: none;
+  }
+
+  .contact-form h1 {
+    font-family: var(--font-family);
+    font-size: var(--font-size-h2);
+    color: #fff;
+    display: flex;
+    justify-content: center;
+  }
+
+  .flex .flex-box h1 {
+    font-family: var(--font-family);
+    font-size: var(--font-size-h2-desktop);
+    color: #091e42;
+  }
+
+  #first-sec h1 {
+    margin-top: 3rem;
+    padding-top: 4rem;
+    font-size: var(--font-size-h1-desktop);
+    font-family: var(--font-family);
+    font-weight: 700;
+    letter-spacing: -3px;
+    color: #172b4d;
+  }
+
+  .contact-form p {
+    font-family: var(--font-family);
+    font-size: 20px;
+    color: #ebebff;
+    width: 40vw;
+    margin: -1rem 8rem 0 8rem;
+    text-align: justify;
+  }
+
+  .desktop-work-sec .div-info p {
+    font-family: var(--font-family);
+    font-size: var(--font-size-p);
+    color: #344563;
+    margin-left: 1.2rem;
+  }
+
+  .flex .flex-box p {
+    font-size: 16px;
+    font-family: var(--font-family);
+    color: #344563;
+  }
+
+  #first-sec p {
+    font-size: var(--font-size-p-desktop);
+    font-family: var(--font-family);
+    text-align: justify;
+    padding-top: -1rem;
+    margin-top: -1rem;
+  }
+
+  .flex .flex-box h3 {
+    font-family: var(--font-family);
+    font-size: 16px;
+    color: #7f8cff;
+  }
+
+  #first-sec h3 {
+    font-family: var(--font-family);
+    font-size: var(--font-size-h3-desktop);
+    color: #7f8cff;
+    text-align: justify;
+    padding-top: -1rem;
+    margin-top: -0.5rem;
+  }
+
+  .contact-form form ul {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem 10vw 1rem 10vw;
+  }
+
+  #first-sec ul {
+    display: flex;
+    margin-left: -3.5rem;
+    padding-top: -1rem;
+    margin-top: -1rem;
+  }
+
+  .div-info .sec-list li {
+    float: left;
+    margin: 1vw;
+    background-color: #ebebff;
+    padding: 0.5rem 1rem 0.5rem 1rem;
+    border-radius: 8px;
+    height: 2vw;
+  }
+
+  .flex .flex-box .social-img li {
+    float: left;
+    margin: 1vw;
+  }
+
+  .flex-3 .grid .grid-2 .language li {
+    float: left;
+    background-color: #f7f7f9;
+    margin: 1vw;
+    font-size: 15px;
+    color: #253858;
+    width: 9vw;
+    height: 9vw;
+    border-radius: 4px;
+  }
+
+  #first-sec ul li {
+    margin: 0.4rem 1rem 0 1rem;
+  }
+
+  .work-sec {
+    display: none;
+  }
+
+  .desktop-work-sec {
+    display: grid;
+    gap: 3.5rem;
+    grid-template-columns: 1;
+    grid-template-rows: repeat(4, 40rem);
```

Comment:
- [ ] Please 🙏 reduce the gap between project cards to adjust your design according to [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0)

**_Figma design_**


<img width="701" alt="Screenshot 2023-02-23 at 9 10 05 PM" src="https://user-images.githubusercontent.com/44798044/220977569-b3123fad-bac4-4f30-ab3b-b7e23be7aae1.png">

_**Your design**_

<img width="1083" alt="Screenshot 2023-02-23 at 9 10 21 PM" src="https://user-images.githubusercontent.com/44798044/220977613-314e9bb4-4385-4026-90f6-f283ac5753af.png">